pkg/cli/start.go

```diff
@@ -330,8 +330,8 @@ func runStart(cmd *cobra.Command, args []string, startSingleNode bool) (returnEr
 	// has completed.
 	// TODO(andrei): we don't close the span on the early returns below.
 	tracer := serverCfg.Settings.Tracer
-	sp := tracer.StartSpan("server start")
-	ctx = tracing.ContextWithSpan(ctx, sp)
+	startupSpan := tracer.StartSpan("server start")
```

Comment:
Why this rename? We call this `sp` everywhere else and there isn't anything special about this one (except that it's long-running).

I think it would be more useful to terminate this span when the server has fully started, and to make another span for the shutdown sequence, by the way. We should minimize spans that are basically always there as they'll just clutter up the registry and will have their events ultimately dropped anyway.